path_config.go

```diff
@@ -34,6 +28,17 @@ extracted.`,
 				Type:        framework.TypeString,
 				Description: "PEM encoded CA cert for use by the TLS client used to talk with the API.",
 			},
+			"token_reviewer_jwt": {
+				Type:        framework.TypeString,
+				Description: "A service account JWT used to access the TokenReview API to validate other JWTs during login.",
```

Comment:
I'd mention that this is optional too, and make it clear one or the other needs to be set.